collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -612,6 +611,11 @@ protected void onActivityResult(int requestCode, int resultCode,
             return;
         }
 
+        if (intent == null) {
+            Timber.i("The intent has a null value for requestCode: " + requestCode);
```

Comment:
I am also in for adding Crashalytics. But it's not fair to keep this issue open until then. Maybe we can decide a better option for right now and then later when Crashalytics is implemented then use the best option at that moment